packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/create-transform-job.ts

```diff
@@ -173,6 +181,15 @@ export class SageMakerCreateTransformJob extends sfn.TaskStateBase {
     };
   }
 
+  private renderModelClientOptions(config: ModelClientOptions): { [key: string]: any } {
+    return {
+      ModelClientConfig: {
+        InvocationsMaxRetries: config.invocationsMaxRetries ?? 0,
+        InvocationsTimeoutInSeconds: config.invocationsTimeout?.toSeconds() ?? 300,
```

Comment:
similar comment here, rather than explicitly setting one, we should fall back to the service defaults? If it's 300, we're only documenting that behaviour but not explicitly setting it

Reply:
Makes sense. I was trying to avoid the corner case when only 1 of them is defined. I can update these values based on what I've observed from practice.

Reply:
is 3 the limit on retries? I was just thinking the > 0 part for validation

Reply:
Yes, 3 is the [limit on the number of retries](https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_ModelClientConfig.html).

> Valid Range: Minimum value of 0. Maximum value of 3.